src/views/v2/components/data.php

```diff
@@ -1,16 +1,16 @@
 <?php
 /**
-* View: Events Data Object.
-*
-* Override this template in your own theme by creating a file at:
-* [your-theme]/tribe/events/views/v2/data.php
-*
-* See more documentation about our views templating system.
-*
-* @link {INSERT_ARTCILE_LINK_HERE}
-*
-* @version 4.9.4
-*/
+ * View: Events Data Object.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/data.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9.4
```

Comment:
This has been changed:

```suggestion
 * @version TBD
```